src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/NeoWrist/NeoWrist.java

```diff
@@ -53,9 +56,24 @@ public void resetAngle(Rotation2d position) {
         motor.getEncoder().setPosition(position.getRadians());
     }
 
+    @Override
+    public void resetEncoder() {
+        resetAngle(Rotation2d.fromRotations(motor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle).getPosition()));
+    }
+
     @Override
     public void moveToAngle(Rotation2d targetAngle) {
-        motor.getPIDController().setReference(targetAngle.getRadians(), CANSparkMax.ControlType.kPosition, NeoWristConstants.PID_SLOT, motor.getPIDController().getFF());
+        motor.getPIDController().setReference(
+                targetAngle.getRadians(),
+                CANSparkMax.ControlType.kPosition,
+                NeoWristConstants.PID_SLOT,
+                NeoWristConstants.WRIST_FEED_FORWARD.calculate(lastInputs.velocity)
+        );
+    }
+
+    @Override
+    public void standInPlace() {
```

Comment:
I think the `standInPlace ` function should also do PID control on the desired position, no?

Reply:
no.

Reply:
but the mechanism will not stay in place... it is subjected to external forces acting on it that will effect it

Reply:
this is outside of the scope of this CR, making standInPlace use PID changes a lot of the code.

Reply:
also at that point you are just doing moveToAngle

Reply:
please bring it up in the weekly tomorrow then